src/action-ros-ci.ts

```diff
@@ -483,46 +484,50 @@ done`;
 	}
 	await execBashCommand(colconBuildCmd, colconCommandPrefix, options);
 
-	// ignoreReturnCode is set to true to avoid having a lack of coverage
-	// data fail the build.
-	const colconLcovInitialCmd = "colcon lcov-result --initial";
-	await execBashCommand(colconLcovInitialCmd, colconCommandPrefix, {
-		...options,
-		ignoreReturnCode: true,
-	});
+	if (!skipTests) {
```

Comment:
This change is fine - but a nitpick: the indentation of this entire block suggests that it could be a separate function.

Reply:
done in eacf631885eed5b33dab64da90341164b04787c9

Other stuff should probably be split out at some point too.